contribute-to-our-docs.rst

```diff
@@ -48,8 +48,41 @@ prevent you from contributing. If something is inconsistent, we'll fix it.
 We'd rather have documentation that we can fix than not have documentation at 
 all.
 
+Open Documentation Academy
+--------------------------
+Launchpad is a proud member of the Canonical Open Documentation Academy (CODA)
+
+The Open Documentation Academy is an initiative that aims to encourage open 
+source contributions from the community by making documentation the gateway, 
+thus lowering the barrier to entry. 
+
+Through the academy, community members get access to curated and well-defined 
+beginner-friendly tasks along with mentorship and advice in a friendly and 
+encouraging environment. This allows them to make meaningful contributions to
+established open-source projects such as Launchpad from the start.
+
+For experts, the academy can also be a great place to share knowledge and get
+involved with new developments. 
+
+For more information on how you can get involved, visit the `CODA website`_. 
+You can also browse through the types of issues you'll be working on in the `CODA repository`_. 
+
+Other channels to get involved with the academy include:
+
+- Our `discussion forum`_ on the Ubuntu Community Hub
```

Comment:
You use "our" for the first time here... I would probably try to have a coherent style and write someting like "Coda's discussion forum".